### ISSUE TITLE
Table borders not rendered for empty JSON values under "Style::modern()" and "collapsed" mode

### ISSUE THREAD

**Issue by limads** [zhiburt/tabled#250]
Hello, this crate has been really helpful in a REPL program I've been writing, thank you.

I'm using `json_to_table` (v 0.2.0 from crates.io). I'm rendering JSON objects using the "Modern" table style, and ran into the following issue:

```rust
use json_to_table::json_to_table;
use serde_json::json;
let value = json!(
    {
        "field1": "Value",
        "field2": 3,
        "field3" : {}
    }
);
let mut table = json_to_table(&value);
table.set_style(tabled::style::Style::modern());
println!("{}", table.to_string());

let mut table_collapsed = table.clone();
println!("{}", table_collapsed.collapse().to_string());
```

The first table renders correctly, but the borders for the second table aren't rendered for the missing "field3":

![Screenshot from 2022-11-10 09-03-14](https://user-images.githubusercontent.com/24863708/201086945-49118f03-e3db-4b47-9709-ca7ea63f5d92.png)

Best,

**Comment by zhiburt** [comment]
Hi @limads

Thank you for report and the example,
And sorry for inconveniences.

I believe it must be fixed in `0.3.0`.
Could you try it out?

**Comment by limads** [comment]
There was no inconvenience at all. Can confirm the problem has been solved, thank you!
